gslib/third_party/storage_apitools/storage_v1_client.py

```diff
@@ -35,16 +35,8 @@ class StorageV1(base_api.BaseApiClient):
   _VERSION = u'v1'
   _CLIENT_ID = 'nomatter'
   _CLIENT_SECRET = 'nomatter'
-  _USER_AGENT = 'apitools gsutil/%s Python/%s (%s)' % (
-      gslib.VERSION, platform.python_version(), sys.platform)
-  if system_util.InvokedViaCloudSdk():
-    _USER_AGENT += ' google-cloud-sdk'
-    if system_util.CloudSdkVersion():
-      _USER_AGENT += '/%s' % system_util.CloudSdkVersion()
-  if MetricsCollector.IsDisabled():
-    _USER_AGENT += ' analytics/disabled'
-  else:
-    _USER_AGENT += ' analytics/enabled'
+  _USER_AGENT = 'apitools Python/%s' % platform.python_version()
+  _USER_AGENT += GetUserAgent(MetricsCollector.IsDisabled())
```

Comment:
Since the UserAgent is going to remain the same throughout the execution of the program, does it make sense to create a generic global constant which stores the result of GetUserAgent when it is first called in `gslib.__main__.py`. The global constant then can be simply appended here as well as appended to the boto.UserAgent above. This way we can avoid parsing the args twice, although the operation is trivial.